src/main/java/no/nav/familie/inntektsmelding/integrasjoner/altinn/AltinnReportee.java

```diff
@@ -0,0 +1,9 @@
+package no.nav.familie.inntektsmelding.integrasjoner.altinn;
+
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+public record AltinnReportee(String name, String organizationForm, String organizationNumber, String parentOrganizationNumber, String socialSecurityNumber, String status, String type) {
+}
```

Comment:
Denne håndtere ikke ukjente props i responsen, så vi bør håndtere det
```suggestion
@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
@JsonIgnoreProperties(ignoreUnknown = true)
public record AltinnReportee(String name, String organizationForm, String organizationNumber, String parentOrganizationNumber, String socialSecurityNumber, String status, String type) {
}
```

Det kan testes med denne
```java

    @Test
    public void mapping_til_AltinnReportee_handterer_ukjent_props() throws IOException {
        String jsonAsString = """
            {
            "name":"a",
            "organizationForm":"something",
            "organizationNumber":"something",
            "parentOrganizationNumber":"something",
            "socialSecurityNumber":"something",
            "status":"something",
            "type":"something",
            "unknown":"property"
            }
            """;
        ObjectMapper mapper = new ObjectMapper();
        AltinnReportee readValue = mapper.readValue(jsonAsString, AltinnReportee.class);

        assertNotNull(readValue);
    }
```

Reply:
Enig i at JSON-konfigurasjon generelt sett bør ligge et felles sted, for å unngå å gjenta det samme flere steder.

I dette tilfellet er Altinn litt spesiell, siden derees navnestandard er å uppercase første bokstav på feltnavnet. For eksempel:

```
{
  "Name": "Saltrød og Høneby",
  "Type": "Business",
  "OrganizationNumber": "999999999",
  "OrganizationForm": "BEDR",
  "Status": "Active",
  "ParentOrganizationNumber": "910825555"
}
```

Det er denne navnestandarden som PropertyNamingStrategies.UpperCamelCaseStrategy.class refererer til. Den bør altså ikke være en del av felles JSON-konfigurasjon. Alternativet til `@JsonNaming` er å definere feltnavn per felt, f. eks. `@JsonProperty("OrganizationNumber")`.

Reply:
Jeg er enig i at JSON-deserialisering bør enhetstestes @sirimykland :)